server/@types/dto/index.d.ts

```diff
@@ -54,20 +69,11 @@ declare module 'dto' {
     howIdentifiedOther?: string
   }
 
-  export interface ChallengeResponseDto {
+  export interface ChallengeResponseDto extends ReferencedAndAuditable {
```

Comment:
Introduced `ReferencedAndAuditable` and used it on the existing `ChallengeResponseDto` and `ConditionDto` types to reduce duplication of the definition of the 9 "reference and audit data fields"
I suspect we will use this on other types in due course ....